delivery/operator/README.md

```diff
@@ -0,0 +1,261 @@
+Framework](https://operatorframework.io) to create and provision a Helm chart-based operator built
+with the [Operator SDK](https://sdk.operatorframework.io). [Operator Lifecycle
+Manager](https://olm.operatorframework.io) is used to deploy the operator.
+
+If you just want to run the tests and review generated artifacts:
+
+1. Put a GitHub username and token with packages:write permissions in your clone of the repo's .env file.
+1. Run `WAIT_FOR_DELETE=1 ./test.sh`.
+
+## Prerequisites
+
+You will need to install [Operator SDK](https://sdk.operatorframework.io/) to
+your workstation and [Operator Lifecycle
+Manager](https://olm.operatorframework.io/) (OLM) to your Kubernetes cluster.
+
+1. Install `operator-sdk` CLI ([official instructions](https://sdk.operatorframework.io/docs/installation/))
+1. Install OLM operator ([official instructions](https://olm.operatorframework.io/docs/getting-started/))
+   
+An example follows. You may also want to review the test script at
+[./test/install-operator-framework.sh](./test/install-operator-framework.sh).
+
+```bash
+OSDK_VERSION=1.16.0
+DEST_DIR=/usr/local/bin
+OS=linux
+ARCH=amd64
+
+curl -L -o "${DEST_DIR}/operator-sdk" \
+    https://github.com/operator-framework/operator-sdk/releases/download/v${OSDK_VERSION}/operator-sdk_${OS}_${ARCH}
+chmod +x "${DEST_DIR}/operator-sdk"
+
+operator-sdk version
+operator-sdk olm install
+operator-sdk olm status
+```
+
+## Deliver
+
+Deliver podtato-head by creating an operator to manage "PodtatoHeadApp"
+resources, then declaring such a resource per the following instructions. You
+may also want to review the test script at
+[./test/build-and-push-operator.sh](./test/build-and-push-operator.sh).
+
+### Build a Helm chart-based operator
+
+In an empty directory init a new Helm operator from the podtato-head chart as follows.
+
+```bash
+chart_dir=./delivery/chart
+# expand relative path so it works from other dirs
+chart_dir=$(cd ${chart_dir} && pwd)
+
+work_dir=./work
+mkdir -p ${work_dir}
+cd ${work_dir}
+
+operator-sdk init --plugins 'helm.sdk.operatorframework.io/v1' --project-version=3 \
+    --project-name 'podtato-head-operator' \
+    --domain=podtato-head.io \
+    --group=apps \
+    --version=v1alpha1 \
+    --kind=PodtatoHeadApp \
+    --helm-chart=${chart_dir}
+```
+
+Now build and push the operator image, the operator bundle, and a catalog which
+includes the bundle as follows. We use GitHub's container registry `ghcr.io`;
+you will need to be signed in to push images. Sign in by getting a personal
+access token with `packages:write` permissions and running the following commands.
+
+```bash
+github_user=
+github_token=
+
+echo "${github_token}" | docker login --username=${github_user} --password-stdin ghcr.io
+
+export IMAGE_TAG_BASE=ghcr.io/${github_user}/podtato-head/operator
+export IMG=${IMAGE_TAG_BASE}:latest
+
+make docker-build docker-push
+make bundle bundle-build bundle-push
+make catalog-build catalog-push
+```
+
+### Deploy the operator
```

Comment:
By using operator-sdk to do that you could just tun the target `make deploy IMG=` or use operator-sdk run bundle <bundle-path> to check it with the OLM integration

Reply:
Thanks! I'll expand the README to mention that as a shortcut.

Reply:
@camilamacedo86 there's a section later in the doc which briefly walks through the make commands including `make deploy`. I updated that section's title to make it clearer. Thanks!